src/Tests/Microsoft.NET.TestFramework/ToolsetInfo.cs

```diff
@@ -57,7 +72,10 @@ public void AddTestEnvironmentVariables(SdkCommandSpec command)
                 command.Environment["MSBuildSDKsPath"] = SdksPath;
                 command.Environment["DOTNET_MSBUILD_SDK_RESOLVER_SDKS_DIR"] = SdksPath;
 
-                command.Environment["MicrosoftNETBuildExtensionsTargets"] = Path.Combine(BuildExtensionsMSBuildPath, "Microsoft.NET.Build.Extensions.targets");
+                //  OK to pass in null as the logger here because SdksPath is set so it won't go down the code path
+                //  that uses the logger
+                var microsoftNETBuildExtensionsPath = GetMicrosoftNETBuildExtensionsPath(null);
```

Comment:
This feels a bit dirty (passing in null for the logger because we know it won't be used).